extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js

```diff
@@ -213,8 +213,17 @@ export class AmpLightboxViewer extends AMP.BaseElement {
         'i-amphtml-element');
     let clonedNode = element.cloneNode(deepClone);
     if (element.tagName == 'FIGURE') {
-      clonedNode = element.getElementsByTagName('amp-img')[0]
-          .cloneNode(deepClone);
+      const ampImg = element.getElementsByTagName('amp-img');
+      user().assert(ampImg.length <= 1,
+          'You cannot have more than one amp-img in a figure');
+      if (ampImg.length == 1) {
+        clonedNode = ampImg[0].cloneNode(deepClone);
+      } else {
+        const img = element.getElementsByTagName('img');
```

Comment:
AMP forbids `img` tags, so unless there's an `<amp-img>`, there won't be an `<img>`

Reply:
Oh that's good to know. In that case I can take out half of this.